docs/input/docs/usage/examples.md

```diff
@@ -0,0 +1,80 @@
+---
+Order: 20
+Title: Examples
+Description: Examples of usages of BBT.Maybe
+---
+
+## Usage of factory methods
+
+For the creation of maybe values factory methods are provided. Methods with arguments for reference are available (Some method). For the none case parameterless methods can be used (None method).
+For nullable value types analogous methods postfixed with 'Struct' are provided.
+
+```csharp
+    // Creation of a maybe for a declared reference type which could be instantiated or not.
+    var foo = new Foo();
+    var maybeFoo = Maybe.Some(foo);
+
+    // Creation of a maybe for a reference type which is null.
+    var maybeFoo2 = Maybe.None<Foo>();
+
+    // Creation of a maybe for a declared nullable value type which could be instantiated or not.
+    int? number = 5;
+    var maybeNumber = Maybe.SomeStruct<int>(number);
+
+    // Creation of a maybe for a nullable value type which is null.
+    var maybeNumber2 = Maybe.NoneStruct<int>();
+```
+
+## Optional properties
+
+Maybe is quite useful for optional properties. The default implementation of maybe represents the null case. Therefore optional properties of type maybe are representing the null case either and must not be initialized.
+
+```csharp
+    /// Data class with mandatory property Foo and optional property Bar. Default of Bar is null.
+    public class Data()
```

Comment:
Remove indentation as this indentates the code block's content.

Reply:
Done.